velox/dwio/common/SelectiveColumnReader.cpp

```diff
@@ -220,6 +220,54 @@ void SelectiveColumnReader::getIntValues(
   }
 }
 
+void SelectiveColumnReader::getUnsignedIntValues(
+    RowSet rows,
+    const TypePtr& requestedType,
+    VectorPtr* result) {
+  switch (requestedType->kind()) {
+    case TypeKind::SMALLINT:
+      switch (valueSize_) {
+        case 4:
+          getFlatValues<int32_t, int16_t>(rows, result, requestedType);
```

Comment:
Should it be `getFlatValues<uint32_t, int16_t>`?

Reply:
updated